src/uct/tcp/tcpcm/tcpcm_iface.c

```diff
@@ -0,0 +1,499 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2017-2019.  ALL RIGHTS RESERVED.
+ * Copyright (C) NVIDIA Corporation. 2019.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#include "tcpcm_iface.h"
+#include "tcpcm_ep.h"
+#include <uct/base/uct_worker.h>
+#include <ucs/sys/string.h>
+#include <ucs/sys/sock.h>
+
+
+enum uct_tcpcm_process_event_flags {
+    UCT_TCPCM_PROCESS_EVENT_DESTROY_SOCK_ID_FLAG = UCS_BIT(0),
+    UCT_TCPCM_PROCESS_EVENT_ACK_EVENT_FLAG       = UCS_BIT(1)
+};
+
+static ucs_config_field_t uct_tcpcm_iface_config_table[] = {
+    {"BACKLOG", "1024",
```

Comment:
Do we need `backlog` parameter? It can be replaced with SOMAXCONN constant.
or if we need this, please, limit backlog with value `min(backlog, SOMAXCONN)`